modules/administration-guide/pages/configuring-single-and-multiroot-workspaces.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: PROCEDURE
+:description: Configuring single and multiroot workspaces
+:keywords: singleroot, multiroot, workspace
+:navtitle: Configuring single and multiroot workspaces
+// :page-aliases:
+
+[id="configuring-single-and-multiroot-workspaces"]
+= Configuring single and multi root workspaces
+
+With the multi-root workspace feature, you can work with multiple project folders in the same workspace. This is useful when you are working on several related projects at once, such as product documentation and product code repositories.
+
+TIP: See link:https://code.visualstudio.com/docs/editor/workspaces[What is a VS Code "workspace"] for better understanding and authoring the workspace files.
+
+[NOTE]
+====
+Starting with {prod-short} 7.73.0, the workspace is set to open in multi-root mode by default.
```

Comment:
@ibuziuk @vitaliy-guliy do you want to keep this information in the doc? we'd have to make it into a tiny excerpt with separate info for upstream and downstream

Reply:
We could omit mentioning the version and leave only the fact that the workspace is set to open in multi-root mode by default.

Reply:
I'm ok to omit it, we just need to mention that multi-root is default